python_modules/dagit/dagit/webapp/src/configeditor/ConfigCodeEditor.tsx

```diff
@@ -126,6 +126,10 @@ injectGlobal`
       width: 100%;
     }
   }
+  .cm-whitespace {
```

Comment:
Let's make a comment in the code that reflects what you said in the commit summary,

Reply:
Sounds good 👍